packages/dds/merge-tree/src/sortedSet.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export abstract class SortedSet<T, U extends string | number> {
```

Comment:
this looks good. i'll let @Abe27342  look at the partial length changes, as he is more familiar with that code